app.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express")
+const { getAllTopics, four0Four } = require("./controllers/topics.controllers")
+const { handleCustomErrors, handleServerErrors } = require("./errors")
+
+const app = express()
+
+app.use(express.json())
```

Comment:
Can we remove this until it's needed? We are currently dealing with any incoming data.